app/src/main/java/com/sun/heartrate/ui/heartbeat/HeartbeatFragment.kt

```diff
@@ -81,11 +85,60 @@ class HeartbeatFragment : Fragment(), HeartbeatContract.View, View.OnClickListen
         }
     }
     
-    private fun upDateIuButton(isCheckCamera: Boolean) {
+    private fun updateUiScreenHeartbeat() {
+        updateUiButton(heartbeatPresenter.checkCamera())
+        updateUiProgressBar(System.currentTimeMillis())
+        updateUiTextViewHeart(0)
+        updateUiTextViewHelp(MIN_ROLLING)
+        updateUiImageViewHeart(heartbeatPresenter.checkCamera())
+    }
+    
+    private fun updateUiButton(isCheckCamera: Boolean) {
```

Comment:
```suggestion
    private fun updateUiButton(isRecording: Boolean) {
```
tên biến k có ý nghĩa, em nên đặt clear hơn